### DISCUSSION TITLE
Cannot connect to Dexcom One transmitter

### DISCUSSION THREAD

**Discussion by reyalex1** [NightscoutFoundation/xDrip#2693] [General]
Hi, as per the title, I cannot connect a Dexcom I've transmitter within xdrip. The transmitter starts with a C. Please see attached system status pages. I have uploaded detailed logs as requested by Navid.
![inbound6762976460823415065](https://user-images.githubusercontent.com/85752598/220549811-5b09a54c-f4bd-4451-be5f-0b244ea95f9b.png)

**Comment by jamorham** [comment]
Can you please update to the latest nightly version and also describe your phone model and android version. Thanks.

**Reply by Navid200** [reply]
I suggested 1 (please see attached image) because I knew the phone was not Xiaomi or OnePlus.  Otherwise, I would have suggested 2.
![Screenshot 2023-02-22 085824](https://user-images.githubusercontent.com/51497406/220642375-739c4e1d-4947-4f55-92a2-1f3f057ea3e3.png)

**Comment by reyalex1** [comment]
Hi
I've removed the sensor and transmitter now and reverted back to using
libre 2 as this was comprising my diabetes management, so updating will not
provide any useful information now, unfortunately. The version installed
was as directed to by Navid.

Phone is a Google pixel 6 with Android 13.

**Comment by reyalex1** [comment]
@Navid200 @jamorham thank you both for the responses. I've uninstalled xdrip and reinstalled, checked all settings, and happy to report that everything is now working!
